### ISSUE TITLE
Ability to Copy Applications / Providers

### ISSUE THREAD

**Issue by obsidiangroup** [goauthentik/authentik#3390]
**Is your feature request related to a problem? Please describe.**
It's frustrating when you have multiple applications that use the exact same settings to have to manually create each one.  The ability to copy applications and providers would solve this.

**Describe the solution you'd like**
The ability to copy applications.  authentik would take the selected applications and copy all settings to the new Application, prompting you for a name.  The prompt could include an option to set the slug, else authentik just uses the application name as the slug.  It should also create the provider

**Comment by AlvinVrolijk** [comment]
Same with flows/stages

**Comment by obsidiangroup** [comment]
Could you export your flow then import it as a new name?

**Comment by AlvinVrolijk** [comment]
If you change the name within the file first, then yeah.On 11 Aug 2022, at 9:48 PM, Darrin ***@***.***> wrote:﻿
Could you export your flow then import it as a new name?

**Comment by BeryJu** [comment]
blocked by #6064

**Comment by dominic-r** [comment]
We're closing this in favor of this parent issue: #25306
